### DISCUSSION TITLE
Weekly Spring 30/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#257] [General]
Ayer hicimos una revisión de todo lo que habíamos hecho: la parte de HTML todavía tiene que ser modificada para agregar el link hacia el sitio de Almoneda cuando lo tengamos hecho, la parte de Python estaba virtualmente terminada, solo faltaba hacer un par de controladores que terminé durante el domingo, y la parte de documentación todavía necesita los casos de uso. Claudia va a empezar a trabajar con el sitio de Almoneda en Wordpress, yo voy a instalarlo en algún servidor y pasarle las claves para que pueda después pasar su trabajo ahí. Los queries están virtualmente terminados también. Falta que algunos miembros elijan de qué van a hablar en el video que vamos a grabar el domingo 6 para que no se pisen con otros. Continuamos la idea de que vamos a grabar el domingo a la mañana y a la noche los que no puedan para tener el video finalizado una semana antes del final del sprint. Cada uno tiene que armar su CV en su página de wordpress para luego subirla al sitio cuando esté disponible.

**Comment by AhumadaCeleste** [comment]
Agregaría, tomar de discussion "Ideas para Javascript #155" e intentar sumar funcionalidades en este lenguaje.
